test/test-utils/client.ts

```diff
@@ -168,5 +168,10 @@ export const mockClientMethodsCrypto = (): Partial<
         isCrossSigningReady: jest.fn().mockResolvedValue(true),
         isSecretStorageReady: jest.fn(),
         getSessionBackupPrivateKey: jest.fn(),
+        checkKeyBackupAndEnable: jest.fn(),
+        isKeyBackupTrusted: jest.fn(),
+        getActiveSessionBackupVersion: jest.fn(),
+        getDeviceVerificationStatus: jest.fn(),
+        getCrossSigningKeyId: jest.fn(),
```

Comment:
can we get rid of any of the old mock methods, like `isKeyBackupStored` and `checkKeyBackup` ?